src/test/scripts/applications/nn/grad_check.dml

```diff
@@ -2537,3 +2538,78 @@ elu = function() {
      }
    }
 }
+
+attention = function() {
```

Comment:
I do not understand what this method is testing, maybe a comment would help.

Reply:
This test is to verify the gradient of the backward pass numerically.
The comments are just copied and modified from the other test cases in this file.

Reply:
The intention of me commenting on it was, that you add a comment in code. When you add it here only i get the help from you.